.ado/jobs/playground.yml

```diff
@@ -64,12 +64,16 @@ jobs:
           script: |
             echo ##vso[task.setvariable variable=BuildWinUI3Properties]/p:AppxPackageSigningEnabled=false /p:UseWinUI3=true
 
-      - task: NuGetCommand@2
-        displayName: NuGet restore - Playground
+      - task: MSBuild@1
+        displayName: Restore NuGet packages
         inputs:
-          command: restore
-          restoreSolution: packages/playground/windows/$(SolutionFile)
-          verbosityRestore: Detailed # Options: quiet, normal, detailed
+          solution: packages/playground/windows/$(SolutionFile)
+          msbuildArchitecture: x64
+          platform: $(BuildPlatform)
+          configuration: $(BuildConfiguration)
+          msbuildArguments:
+            /t:Restore
```

Comment:
This entire step can be removed as it could be done in the main build (VsBuild@1 task) by passing `/restore` (like we already do in a few other places) Msbuild will run a `/t:restore` itself automatically without two invocations.

Reply:
Note: I'm okay to do this later as well, since I really don't want to hold up this goodness. It has been on my cleanup list for a while :D

Reply:
I think this might actually do the whole build

Reply:
Oh or is this to run just the restore target?

Reply:
@dannyvv I'm aware, just wanted to shave some time off the actual build task, to keep those from timing out.
It also gives some separation in case there is a dependency declaration or repository connection glitch, which are conceptually unrelated to actual build errors.

Just doing `/restore` is the smart way to go on a dev machine, of course.